dodola/services.py

```diff
@@ -244,6 +245,24 @@ def remove_leapdays(x, out):
     storage.write(out, noleap_ds)
 
 
+@log_service
+def correct_wet_day_frequency(x, out, process):
+    """Corrects wet day frequency in a dataset
+
+    Parameters
+    ----------
+    x : str
+        Storage URL to input xr.Dataset that will be regridded.
+    out : str
+        Storage URL to write regridded output to.
+    process : {"pre-process", "post-process"}
+        Step in pipeline, used in determining how to correct.
```

Comment:
Either/both in `dodola/services.py` or `dodola/core.py`, it would be nice to elaborate a  little bit more on what this 'process' arg does or when it's used. What's the how and why? As a user I'm thinking "is this a clean step thing (it's a 'correction')? or when do each of these pre/post steps occur?"

Reply:
Thanks for this, adding to `dodola/services.py`.